views/case-summary-record.njk

```diff
@@ -92,6 +92,13 @@
                 <p class="govuk-body">Not active</p>
             {% endif %}
 
+            {% for order in activeOrders %}
```

Comment:
If there are multiple active orders with pre-sentence reports, this will display multiple last pre-sentence reports in the UI.

We'll need to find the last pre-sentence report from however many there may be and display only that one.

Reply:
Actually; I think that there could be a scenario where the last pre-sentence report could actually be in a terminated order.

If they have a long-term order and they re-offend, then they're given a different order that expires before their existing one... then that Pre-Sentence Report would actually be the last one.

Reply:
Ah... after speaking with Charan there might actually be no Pre-Sentence Report on a current order but there could be one on a previous order.

So... we do need to run through all orders to find the actual Last Pre-Sentence Report.

Reply:
I did think about this but then when I looked at the ticket for the community-api implementation, by default it should only return the last found PSR on the probation record checking current events and reports delivered in the last year.

Reply:
After speaking with Chris the date filter to return the last PSR is now going to be handled in the client, so I'll look at this now 🙂